extensions/amp-consent/0.1/consent-config.js

```diff
@@ -320,15 +322,21 @@ export class ConsentConfig {
  * Expand consent endpoint url
  * @param {!Element|!ShadowRoot} element
  * @param {string} url
+ * @param {Object<string, *>=} opt_vars
  * @return {!Promise<string>}
  */
-export function expandConsentEndpointUrl(element, url) {
+export function expandConsentEndpointUrl(element, url, opt_vars) {
   return Services.urlReplacementsForDoc(element).expandUrlAsync(
     url,
     {
       'CLIENT_ID': getConsentCID(element),
+      'CONSENT_PAGE_VIEW_ID_64': () =>
+        getServicePromiseForDoc(element, 'consentStateManager').then(
+          (consentStateManager) => consentStateManager.consentPageViewID64()
+        ),
+      ...opt_vars,
```

Comment:
Shouldn't this be `{...vars, 'CLIENT_ID': ..., 'CONSENT_PAGE_VIEW_ID_64': ...}`?